composer.json

```diff
@@ -41,9 +40,10 @@
         "rector/rector": "^0.18.10",
         "roave/infection-static-analysis-plugin": "^1.16",
         "spatie/phpunit-watcher": "^1.23",
-        "yiisoft/yii-debug": "dev-master",
         "vimeo/psalm": "^4.30|^5.8",
-        "yiisoft/test-support": "^3.0"
+        "yiisoft/test-support": "^3.0",
+        "yiisoft/yii-console": "^2.0",
```

Comment:
If replace `ExitCode` in tests, then we can remove `yii-console` dependency from "require-dev" also.

Reply:
done.